src/main/docker/jenkins/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jenkins/jenkins:lts
+FROM jenkins/jenkins:2.277.4-lts
```

Comment:
Why not use lts-jdk8? This would track also future lts instances?

Reply:
Thank you for your remark. 

I considered this initially but came to the conclusion that the disadvantages outweigh the advantages since there are many steps in the dockerfile that might not be compatible with future updates of the base images. So, the lts-tag is a moving target; this could lead to problems like the one I encountered. I thought it would be more valuable to have a setup guide that is reliably working than to have it following the newest jenkins releases, since I assumed the target audience of this guide to be people that are new to the project and want to onboard quickly,  maybe not with the deepest knowledge of docker.

But anyhow, I am new to the project and if you think the lts-jdk8 is the better choice here, I am happy to update the PR.

Reply:
I would agree on @snowball77 

Additionally, I would prefer to change this to `lts` as soon as https://github.com/ls1intum/jenkins-server-notification-plugin/pull/15 got merged. This will ensure that Artemis is compatible with current versions of its connected components.

Reply:
I just merged ls1intum/jenkins-server-notification-plugin#15 and would agree to @dfuchss to change it to `lts`

Reply:
lts-jdk8 does not solve the problem, the errors are the same (e. g. : 
`E: Unable to locate package libubsan0
ERROR: Service 'jenkins' failed to build: The command '/bin/sh -c apt install -y gcc gdb make libubsan0 liblsan0 libtsan0' returned a non-zero code: 100`

There must have been some changes made to the base image that cause our docker builds to break. So, pinning to the working version jenkins/jenkins:2.277.4-lts seems to be the best option for a quick fix here, imho.

Reply:
In other words: In which cases do I need/use maven, python3.6 and gcc libraries pre-installed? I took a look at the `src/main/tempates/jenkins` and only found Jenkinsfiles with

```
agent {
            docker {
                image '#dockerImage'
                label 'docker'
            }
        }
```

Reply:
I changed it to the `jenkins/jenkins:lts` image.